CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.1.1 (Dec 27, 2020)
+
+* Code cleanup (#441, #442)
+* Implement k-tree with force-k modification (#436, #443)
```

Comment:
#443 is actually a bugfix that's not related to force-k. We had this bug for a long time.

Reply:
> fix(kbucket): put node in the correct position